IPython/core/display.py

```diff
@@ -607,9 +620,19 @@ def data(self, svg):
             pass
         svg = cast_unicode(svg)
         self._data = svg
+    
+    def _data_and_metadata(self):
+        """shortcut for returning metadata with shape information, if defined"""
+        md = {}
+        if self.metadata:
+            md.update(self.metadata)
```

Comment:
Yes that's seem to be the same. I'm also unsure if a helper method to have `_repr_svg_` be one line is necessary.